gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinHelixJobScheduler.java

```diff
@@ -141,6 +143,10 @@ public GobblinHelixJobScheduler(Properties properties,
                                                   metricsWindowSizeInMin);
 
     this.startServicesCompleted = false;
+
+    this.helixJobStopTimeoutSeconds = ConfigUtils.getLong(ConfigUtils.propertiesToConfig(properties),
```

Comment:
probably not required to change at this point but i see this usage of ConfigUtils.propertiesToConfig over an over as if its O(1). may be it needs separate PR for reusing the created config.

Reply:
Its an easy change.